src/atomic-browser.md

```diff
@@ -0,0 +1,97 @@
+You can also fetch your data now as various formats.
+
+Try checking out the other features in the menu bar, and check out the `collections`.
+
+Again, check out the [README](https://github.com/atomicdata-dev/atomic-data-rust/blob/master/server/README.md) for more information and guides!
+
+But first, understand the two Views of the Browser.
+
+## Understanding the two Views of the Browser
+The Atomic Browser has two views
+
+* Edit View `(Ctrl+E)`
+* Data View `(Ctrl+D)`
+
+After installation your browser will be in `Edit View`, also known as 'Data Definitions' View. This means you will be viewing, browsing and editing the Data Definitions.
+A simple `Ctrl+D` will switch you to `Data View` which will hide the data definitions but instead make all the data occurrences visible for viewing, browsing and editing. Just use `(Ctrl+E)` to switch back.
+
+Now, we are ready to create some data.
+
+## Creating your first Atomic Data
+
+Now, in Edit View, let's create a [_Class_](https://atomicdata.dev/classes/Class).
+A Class represents an abstract concept, such as a `BlogPost` (which we'll do here).
+We can do this in a couple of ways:
+
+- Press the `+ icon` button on the left menu (only visible when logged in), and selecting Class
+- Opening [Class](https://atomicdata.dev/classes/Class) and pressing `new class`
+- Going to the [Classes Collection](https://atomicdata.dev/classes/) and pressing the plus icon
+
+The result is the same: we end up with a form in which we can fill in some details.
+
+Let's add a shortname (singular, lower case), and then a description.
+
+After that, we'll add the `required` properties.
+This form you're looking at is constructed by using the `required` and `recommended` Properties defined in `Class`.
+We can use these same fields to generate our BlogPost resource!
+Which fields would be required in a `BlogPost`?
+A `name`, and a `description`, probably.
+
+So click on the `+ icon` under `requires` and search for these Properties to add them.
+
+Now, we can skip the `recommended` properties, and get right to saving our newly created `BlogPost` class.
+So, press save, and now look at what you created.
+
+Notice a couple of things:
+
+- Your Class has its own URL.
+- It has a `parent`, shown in the top of the screen. This has impact on the visibility and rights of your Resource. We'll get to that [later in the documentation](./hierarchy.md).
+
+Now, go to the navigation bar, which is by default at the bottom of the window. Use its context menu to open the `Data View`.
+This view gives you some more insight into your newly created data, and various ways in which you can serialize it.
+
+## How can I see the members of a collection?
+
+The members of a <a href="https://docs.atomicdata.dev/schema/collections.html" target="_blank">collection</a> are the actual data of that collection, and thus you need to be in [the Data View](#understanding-the-two-views-of-the-browser) `(Ctrl+D)` to see them.
```

Comment:
use markdown syntax for links

`[collection](https://docs.atomicdata.dev/schema/collections.html)`